django_spanner/introspection.py

```diff
@@ -225,17 +225,17 @@ def get_constraints(self, cursor, table_name):
         """
         constraints = {}
         quoted_table_name = self.connection.ops.quote_name(table_name)
+        schema_name = self._get_schema_name(cursor)
 
         # Firstly populate all available constraints and their columns.
         constraint_columns = cursor.run_sql_in_snapshot(
-            '''
+            """
             SELECT
                 CONSTRAINT_NAME, COLUMN_NAME
             FROM
                 INFORMATION_SCHEMA.CONSTRAINT_COLUMN_USAGE
-               WHERE TABLE_NAME="{table}"'''.format(
-                table=quoted_table_name
-            )
+               WHERE TABLE_NAME=@table AND TABLE_SCHEMA=@schema_name""",
+            params={"table": quoted_table_name, "schema_name": schema_name},
```

Comment:
We should not use a quoted table name when it is used as a query parameter. Also; as there is no test failing due to this, I think it means that this method is also not covered by any tests. Would you mind adding a test for this method as well?

Reply:
Fixed as discussed